.circleci/config.yml.in

```diff
@@ -152,15 +152,6 @@ commands:
           args: --no-build-isolation <<# parameters.editable >> --editable <</ parameters.editable >> .
           descr: Install torchvision <<# parameters.editable >> in editable mode <</ parameters.editable >>
 
-  install_prototype_dependencies:
```

Comment:
thanks Philip, Do you mind just leaving that one last time, to make sure CI is green (and then remove it)

Reply:
You commented on the CircleCi config, but these test continue to run. We only removed the dependencies there. You meant that I should run the datasets tests again, right?

Reply:
I've temporarily reinstated the prototype datasets CI in 215fb185cf6be5be7adf0388116c77acc9a5d3f3. LMK if that is not what you wanted.

Reply:
CI is red, but for an unrelated failure on the transforms tests. Datasets tests are [green](https://github.com/pytorch/vision/actions/runs/3443182196/jobs/5744471832#step:11:110).